### PR TITLE
Make warning about using performance tuning more explicit

### PR DESCRIPTION
**Pull request by gdubicki**
partially fixes https://github.com/scylladb/scylla-operator/issues/2017

### CODE REVIEW

File: docs/source/performance.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 Scylla Operator 1.6 introduces a new experimental feature allowing users to optimize Kubernetes nodes.
 
+:::{warning}
+The tool performing the tuning doesn't support reverting the changes (see [scylladb/seastar#2350](https://github.com/scylladb/seastar/issues/2350)).
+
+The only complete way to revert them to disable the tuning **and** restart all the nodes hosting your cluster.
+
+There also were cases of performance degradation after applying the tuning (f.e. [scylladb/scylla-operator#2017](https://github.com/scylladb/scylla-operator/issues/2017)).
```
Comment (tnozicka) [inline docs/source/performance.md:10]: referencing one *open* issue that has not yet been even triaged, doesn't fit a general claim made here (`were case`)

**Review by tnozicka — COMMENTED** [review]
I think a general `:::note` to say that you should try performance tuning first on your staging cluster is ok, although that should be done for every change with any software when you care about your production availability. I'd avoid docs content based on issues that have just been opened / the colored wording

**Comment by gdubicki** [comment]
On second thought, I agree with you, @tnozicka. I was a bit emotional when I wrote this as I had spent most of the weekend fixing our cluster…

Can you just please reword the warning in this PR as you see fit?

**Comment by tnozicka** [comment]
np, how about something like this:
```
:::{note}
We recommend that you first try out the performance tuning on a pre-production instance.
Given the nature of the underlying tuning script, undoing the changes requires rebooting the Kubernetes node(s).
:::
```

**Review by tnozicka — APPROVED** [review]
/approve
/lgtm
thanks

**Comment by tnozicka** [comment]
/hold
please squash the commits

**Comment by gdubicki** [comment]
> /hold please squash the commits

Done! 😊

**Comment by tnozicka** [comment]
/lgtm
thanks

**Pull request merged**